docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,428 @@
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_ad7616_dma  13  57         89
+spi_ad7616 **   12  56         88
+axi_ad7616 *    10  54         87
+=============== === ========== ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here <main:/>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the **\*PROJECT LOCATION WITHIN HDL (EX:
+projects/ad7616_sdz/zed)**\ \* location and run the make command by
+typing in your command prompt:
+
+**Linux/Cygwin/WSL**
+
+**\*Say which is the default configuration that's built when running
+``make``, give examples of running with all parameters and also with
+just one. Say that it will create a folder with the name ... when
+running with the following parameters.**\ \*
+
+.. code-block::
+   :linenos:
+   :emphasize-lines: 2, 6
+
+   user@analog:~$ cd hdl/projects/ad7616_sdz/zed
+   // these are just examples of how to write the *make* command with parameters
+   user@analog:~/hdl/projects/ad7616_sdz/zed$ make parameter1=value
+
+   user@analog:~$ cd hdl/projects/ad7616_sdz/zed
+   // these are just examples of how to write the *make* command with parameters
+   user@analog:~/hdl/projects/ad7616_sdz/zed$ make SER_PAR_N=0
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make SER_PAR_N=0``
+
+then the folder name will be:
+
+``SERPARN0``
+The name will not exceed the limits of the Operating System of 260 characters.
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Connections and hardware changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   **The following hardware changes are required:**
+
+   (**Please note:** Because of the **SDP-I-FMC** the level of the **VADJ** in
+   the carrier board needs to be set to **3.3V**.
+
+   Depending on the required interface mode, some hardware modifications need to
+   be done.
+
+   -   **SL5** - unmounted - Parallel interface
+   -   **SL5** - mounted - Serial interface
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :dokuwiki:`AD7616 User guide <resources/eval/user-guides/ad7616-sdz>`
+-  :dokuwiki:`How to build No-OS <resources/no-os/build>`
```

Comment:
The No-OS documentation should be placed under ``Software related`` section

Reply:
Done.